Product/SoapUI_Test/RegressionSuite/Passthru/DocQueryRetrieveSelfTest/DocQueryRetrieveSelfTest-soapui-project.xml

```diff
@@ -8078,7 +8078,7 @@ assert docCount.toInteger() == 0;</scriptText>
       </con:testStep>
       <con:properties/>
       <con:reportParameters/>
-      <con:tearDownScript>FileUtils.restoreToMasterConfiguration(context, log);</con:tearDownScript>
+      <con:tearDownScript>FileUtils.restoreConfiguration(context.findProperty("GatewayPropDir"), log);</con:tearDownScript>
```

Comment:
Does it make any difference after changing function name in Teardown script for this test case ? , because this test case is disabled originally.